ocs_ci/ocs/node.py

```diff
@@ -104,6 +106,14 @@ def get_nodes(node_type=constants.WORKER_MACHINE, num_of_nodes=None):
             if node_type
             in node.ocp.get_resource(resource_name=node.name, column="ROLES")
         ]
+    if is_hci_provider_cluster() and node_type == constants.WORKER_MACHINE:
+        typed_nodes = [
```

Comment:
Wouldn't it be better to describe this new behaviour in the function's docstring? It might not be obvious to the people who are going to use it.

Reply:
Ack, I will modify the docstring.